src/components/legend/attributes.js

```diff
@@ -79,12 +79,12 @@ module.exports = {
     },
     itemsymbol: {
         valType: 'enumerated',
-        values: ['trace', 'constant'],
+        values: ['trace', 'circle'],
```

Comment:
Won't be better to use `auto` instead of `trace` here?
@alexcjohnson @nicolaskruchten

Reply:
Well "trace" is basically doing what it does for `itemsizing`. For `itemsizing`, it uses the size of the first point in the trace. So for `itemsymbol`, I would think you'd want to follow the same pattern?

Reply:
OK. Let's the `trace`.